src/stylesheets/core/_functions.scss

```diff
@@ -900,28 +900,61 @@ Derive a color from a color shortcode
 ----------------------------------------
 */
 
-@function color($shortcode) {
-  $shortcode: smart-quote(unpack($shortcode));
-  @if not $shortcode {
-    @return false;
+@function color($value, $flags...) {
+  $value: unpack($value);
+
+  // Non-token colors may be passed with specific flags
+  @if type-of($value) == color {
+    // override or set-theme will allow any color
+    @if index($flags, override) or index($flags, set-theme) {
+      // override + no-warn will skip warnings
+      @if index($flags, no-warn) {
+        @return $value;
+      }
+
+      @if $theme-show-compile-warnings {
+        @warn 'Override: `#{$value}` is not a USWDS color token.';
+      }
+
+      @return $value;
+    }
   }
-  @if map-has-key($system-color-shortcodes, $shortcode) {
-    $our-color: map-get($system-color-shortcodes, $shortcode);
-    @if $our-color == false {
-      @error '`#{$shortcode}` is a color that does not exist '
-        + 'or is set to false.';
+
+  // False values may be passed through when setting theme colors
+  @if $value == false {
+    @if index($flags, set-theme) {
+      @return $value;
```

Comment:
Rewrote this to restrict the `false` pass-through to `set-theme` only — this allows the `color()` function to be used when setting the theme token (since theme tokens can be `false`).